apps/passport-client/src/dispatch.ts

```diff
@@ -330,8 +330,8 @@ export async function dispatch(
       return hideEmbeddedScreen(state, update);
     case "zapp-connect":
       return zappConnect(state, update, action.zapp, action.origin);
-    case "approve-zapp":
-      return approveZapp(state, update);
+    case "zapp-approval":
```

Comment:
why change this?

Reply:
Previously "approve-zapp" meant "approve this zapp". Now "zapp-approval" means "set the zapp-approval state to this boolean value", meaning that it can also cover _not_ approving the Zapp. Possibly "set-zapp-approval" might be a clearer name.